proto/storage/compliance_operator_v2.proto

```diff
@@ -41,16 +41,16 @@ message ProfileShim {
 
 // Next Tag: 13
 message ComplianceOperatorProfileV2 {
-    string              id              = 1 [(gogoproto.moretags) = 'sql:"pk"'];
-    string              profile_id      = 2;
-    string              name            = 3 [(gogoproto.moretags) = 'search:"Compliance Profile Name,hidden,store" sql:"index=category:unique;name:rule_unique_indicator"'];
-    string              version         = 4 [(gogoproto.moretags) = 'search:"Compliance Operator Version,hidden,store" sql:"index=category:unique;name:rule_unique_indicator"'];
-    string              profile_version = 5;
-    repeated string     product_type    = 6 [(gogoproto.moretags) = 'search:"Compliance Operator Profile Product Type,hidden,store"'];
-    string              standard        = 7 [(gogoproto.moretags) = 'search:"Compliance Operator Profile Standard,hidden,store"'];
-    map<string, string> labels          = 8;
-    map<string, string> annotations     = 9;
-    string              description     = 10;
+    string              id               = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    string              profile_id       = 2;
+    string              name             = 3 [(gogoproto.moretags) = 'search:"Compliance Profile Name,hidden,store" sql:"index=category:unique;name:profile_unique_indicator"'];
+    string              operator_version = 4;
+    string              profile_version  = 5 [(gogoproto.moretags) = 'search:"Compliance Operator Profile Version,hidden,store" sql:"index=category:unique;name:profile_unique_indicator"'];
```

Comment:
In preparation for the profile version being added to the profile from the compliance operator team.